yamcs-core/src/main/java/org/yamcs/cfdp/CfdpUtils.java

```diff
@@ -23,10 +26,18 @@ public static short getUnsignedByte(ByteBuffer buffer) {
     }
 
     /*
-     * write the given short as an unsigned byte to the given buffer at its current position
+     * write the given int as an unsigned byte to the given buffer at its current position
      */
     public static void writeUnsignedByte(ByteBuffer buffer, int input) {
-        buffer.put((byte) (input & 0xff));
```

Comment:
I think you can just use 
buffer.put((byte)input)

the input &0xFF is quite "idiomatic" in Java (i.e. you could see it all over the place and is optimized by the compiler) but you don't need it here I believe